public/spec/features/read_more_spec.rb

```diff
@@ -0,0 +1,134 @@
+require 'spec_helper'
+require 'rails_helper'
+
+describe 'Read More', js: true do
```

Comment:
It would be good to an example with multiple notes of the same type Biographical / Historical (note_multipart).